Checkpoint1/Checkpoint1.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Checkpoint1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int selector = 0;
+            Console.WriteLine("Please select which part you would like to try");
+            Console.WriteLine("Enter 1, 2, 3, 4, or 5");
+            selector = Convert.ToInt32(Console.ReadLine());
+            if (selector == 1)
+            {
+               problemOne(); 
+            }
+            else if (selector == 2)
+            {
+                problemTwo();
+            }
+            else if (selector == 3)
+            {
+                problemThree();
+            }
+            else if (selector == 4)
+            {
+                problemFour();
+            }
+            else if (selector == 5)
+            {
+                problemFive();
+            }
+        //1) count numbers between 1-100 are divisible by 3 with no remainder. Display to console
+
+        //2) Continuously ask user to enter a number or "ok" to exit. Calc the sum of all numbers and display to console
+
+        //3) Ask user to enter a number, compute the factorial of the number and print to console.
+
+        //4) Program picks a random number between1-10, give user 4 chances to guess. If guessed, display "You won!", else, "You lost".
+
+        //5) Ask user to enter a series of numbers separated by commas. Find biggest number and display to console.
+
+        }
+        static void problemOne()
+        {
+            int total = 0;
+            for (int i = 0; i < 100; i++)
+            {
+                if ( i % 3 == 0)
+                {
+                    total++;
+                    Console.WriteLine(i);
+                }
+            }
+            Console.WriteLine("Total: {0}", total);
+            return;
+        }
+        static void problemTwo()
+        {
+            string end = "ok";
+            List<int> two = new List<int>();
+            Console.WriteLine("Please enter a number: ");
+            two.Add(Convert.ToInt32(Console.ReadLine()));
+            while (true)
+            {
+                Console.WriteLine("Please enter another number or type 'ok' to end: ");
+                if ( Console.ReadLine() == end )
```

Comment:
You need to capture this `Console.ReadLine()` in a variable, so on line 77 you can Add it in the list if it is not `ok`